tests/unit/cred-revocation.test.js

```diff
@@ -1,31 +1,34 @@
-import { isRevocationCheckNeeded } from '../../src/utils/vc';
+import {
+  isRevocationCheckNeeded,
+  expandedStatusProperty
+} from '../../src/utils/vc';
 
 describe('Check isRevocationCheckNeeded', () => {
   test('isRevocationCheckNeeded returns true when credentialStatus is present and forceRevocationCheck is true and revocation API is not given', () => {
-    expect(isRevocationCheckNeeded({}, true, undefined)).toBe(true);
+    expect(isRevocationCheckNeeded({ [`${expandedStatusProperty}`]: {} }, true, undefined)).toBe(true);
   });
 
   test('isRevocationCheckNeeded returns true when credentialStatus is present and forceRevocationCheck is true and revocation API is given', () => {
-    expect(isRevocationCheckNeeded({}, true, { dock: 'Anything' })).toBe(true);
```

Comment:
Hmm. Seems like some of these tests were incorrect before as they weren't passing objects with a credentialStatus property when they claimed to do so.

Good that you corrected them.